pkg/policies/controlplane/policy-service.go

```diff
@@ -289,6 +291,73 @@ func (s *PolicyService) PostDynamicConfig(ctx context.Context, req *policylangv1
 	return new(emptypb.Empty), nil
 }
 
+// GetDynamicConfig gets dynamic config of a policy.
+func (s *PolicyService) GetDynamicConfig(ctx context.Context, req *policylangv1.GetDynamicConfigRequest) (*policylangv1.GetDynamicConfigResponses, error) {
+	etcdPolicy, err := s.etcdClient.Client.KV.Get(
+		ctx,
+		path.Join(paths.PoliciesAPIConfigPath, req.PolicyName),
+		clientv3.WithKeysOnly(),
+	)
+	if err != nil {
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+
+	if len(etcdPolicy.Kvs) == 0 && s.policyFactory.GetPolicyWrapper(req.PolicyName) == nil {
+		return nil, status.Error(codes.NotFound, "no such policy")
+	}
+
+	resp, err := s.etcdClient.KV.Get(ctx, path.Join(paths.PoliciesAPIDynamicConfigPath, req.PolicyName))
+	if err != nil {
+		return nil, fmt.Errorf("failed to write dynamic config '%s' to etcd: '%s'", req.PolicyName, err)
+	}
+
+	configs := []*policylangv1.GetDynamicConfigResponse{}
+	for _, v := range resp.Kvs {
```

Comment:
I don't think we would be getting multiple values here? We only store one DynamicConfig per policy and overwrite it on next apply.